core/app.js

```diff
@@ -45,3 +56,34 @@ app.listen(ENV.PORT)
 console.log("Server running on port", ENV.PORT);
 console.log("Development:", !!ENV.HLRDESK_DEV);
 console.log("Hostname:", ENV.HLRDESK_HOST);
+
+
+
+
+
+var sock = require('express')();
+var http = require('http').Server(sock);
```

Comment:
It isn't necessary to create a whole new server here; sockets will run just fine on the koa server we already have set up.